public/style.css

```diff
@@ -27,6 +27,7 @@ h1 {
   text-align:center;
 }
 
+/*Uppercase color orange*/
```

Comment:
what do you mean with uppercase? Maybe add a comment /* orange */ after `color:#e94e1b;` if that’s what makes it easier to read?